src/telemetry/Manager.cc

```diff
@@ -135,6 +135,10 @@ void Manager::InitPostScript() {
                               []() { return static_cast<double>(get_stats()->fds); });
 #endif
 
+    // These two metrics get set at startup and are never modified after.
+    process_start_time = GaugeInstance("process", "start_time", {}, "Process start time", "seconds");
```

Comment:
Just wondering whether one would ever need the timezone of this value, but probably not. The actual value of this is not really interesting, and its only function is to be able to see it change.